### DISCUSSION TITLE
Name Ideas

### DISCUSSION THREAD

**Discussion by tuckplace** [cs-utulsa/Encrypted-Chat-Service#44] [Ideas]
Need ideas for names in this thread. Please suggest any names you come up with and share them here. Once we have a collected list, we can have a vote.

**Comment by tuckplace** [comment]
Name idea: Hermes - the Greek messenger god.

**Comment by mypleasure370** [comment]
Mercury - the Roman equivalent of hermes

**Comment by mypleasure370** [comment]
Anubis - the Egyptian equivalent of hermes
